test/unit/testBraveAlerter/testBraveAlerterSendReminderMessageForSession.js

```diff
@@ -2,236 +2,223 @@ const chai = require('chai')
-                toPhoneNumber: '+11231231234',
-                fromPhoneNumber: '+11231231234',
-                reminderMessage: 'My message',
-            })
-        })
-
-        afterEach(function() {
-            this.braveAlerter.getAlertSession.restore()
-            Twilio.sendTwilioMessage.restore()
-        })
-
-        it('should not call the callback', function() {
-            expect(this.fakeAlertSessionChangedCallback).not.to.be.called
-        })
-
-        it('should log the error', function() {
-            expect(helpers.log).to.be.calledWith('Failed to send reminder message for session guid-123')
-        })
-    })
-
-    describe('if AlertSession is not started', function() {
-        beforeEach(async function() {
-            // Don't actually call Twilio
-            sinon.stub(Twilio, 'sendTwilioMessage').returns({})
-            
-            // Spy on the alertSessionChangedCallback call
-            this.fakeAlertSessionChangedCallback = sinon.fake()
-            
-            this.braveAlerter = new BraveAlerter(
-                dummyGetAlertSession,
-                dummyGetAlertSessionByPhoneNumber,
-                this.fakeAlertSessionChangedCallback,
-            )
-
-            sinon.stub(this.braveAlerter, 'getAlertSession').returns(new AlertSession(
-                'guid-123',
-                'not ALERT_STATE.STARTED',
-            )),
-            
-            await this.braveAlerter.sendReminderMessageForSession({
-                sessionId: 'guid-123',
-                toPhoneNumber: '+11231231234',
-                fromPhoneNumber: '+11231231234',
-                reminderMessage: 'My message',
-            })
-        })
-
-        afterEach(function() {
-            this.braveAlerter.getAlertSession.restore()
-            Twilio.sendTwilioMessage.restore()
-        })
-            
-        it('should not send the reminder', function() { 
-            expect(Twilio.sendTwilioMessage).not.to.be.called
-        })
-
-        it('should not call the callback', function() {
-            expect(this.fakeAlertSessionChangedCallback).not.to.be.called
-        })
+function dummyGetAlertSession() {
+  return 'getAlertSession'
+}
+
+function dummyGetAlertSessionByPhoneNumber() {
+  return 'getAlertSessionByPhoneNumber'
+}
+
+describe('braveAlerter.js unit tests: sendReminderMessageForSession', () => {
+  beforeEach(() => {
+    // Don't actually log
+    sinon.stub(helpers, 'log')
+  })
+
+  afterEach(() => {
+    helpers.log.restore()
+  })
+
+  describe('if AlertSession is started', () => {
+    beforeEach(async () => {
+      // Don't actually call Twilio
+      sinon.stub(Twilio, 'sendTwilioMessage').returns({})
+
+      // Spy on the alertSessionChangedCallback call
+      this.fakeAlertSessionChangedCallback = sinon.fake()
+
+      this.braveAlerter = new BraveAlerter(dummyGetAlertSession, dummyGetAlertSessionByPhoneNumber, this.fakeAlertSessionChangedCallback)
```

Comment:
not sure I understand the linter behaviour in some of these situations, here, the linter has changed this variable assignment to have all parameters on the same line, [elsewhere](https://github.com/bravetechnologycoop/brave-alert-lib/pull/6/files#diff-6d231be596687d632ffb3796e5f4b18535c084f53d2762057f0c11de9adc6c8aR164) the opposite seems to be happening. Is this because we're calling a constructor in this case?

that said, overall looks fine and I don't have a reason not to approve

Reply:
Thanks @mariocimet . My guess would be it depends on the length of the line. If it can fit it all on one, then it will. If not, then it will break it up to the max. But yeah, it's not what I as a human would do, lol.